app/app.go

```diff
@@ -698,6 +702,15 @@ func (app *App) setPostHanders() {
 	app.SetPostHandler(postHandler)
 }
 
+func (*App) ForbiddenParams() [][2]string {
```

Comment:
[Non blocking][Question]
Do we have a discussion where the forbidden params are chosen?